.github/workflows/master.yml

```
@@ -38,9 +38,9 @@ jobs:
           ./scripts/dev-env/run.sh ./scripts/build_a32nx.sh --no-tty -j 4
       - name: Build ZIP files
         run: |
-          ./scripts/dev-env/run.sh node ./scripts/fragment.js
+          ./scripts/dev-env/run.sh node ./scripts/fragment_a32nx.js
           cp ./fbw-a32nx/out/build-modules/modules.json ./fbw-a32nx/out/flybywire-aircraft-a320-neo/install.json
-          ./scripts/dev-env/run.sh node ./scripts/install-source.js
+          ./scripts/dev-env/run.sh node ./scripts/install-source-a32nx.js
```

```suggestion
          ./scripts/dev-env/run.sh node ./scripts/install-source_a32nx.js
```

Could you use an "_" to be in line with the other scripts?